src/psyclone/psyGen.py

```diff
@@ -5552,17 +5557,19 @@ class Symbol(object):
     :raises NotImplementedError: Provided parameters are not supported yet.
     :raises TypeError: Provided parameters have invalid error type.
     :raises ValueError: Provided parameters contain invalid values.
+
```

Comment:
It is what emacs does by default so I've been following it. I agree that it would be better to be coherant. Let's discuss this at the next PSyclone telco.